contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -51,7 +51,7 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
     // A monotonically incremented integer tracking the total number of messages sent by this TeleporterMessenger contract.
     // Used to provide uniqueness when generating message IDs for new messages. Initially starts at 1 such that the
```

Comment:
We should update this comment to destinguish between initialization and first usage of this variable. Perhaps something like: "The first message sent will have a `messageNonce` of 1 such that the nonce value can be used to provide replay protection"